.rhdh/scripts/prepare-restricted-environment.sh

```diff
@@ -13,14 +13,14 @@ NC='\033[0m'
 IS_OPENSHIFT=""
 IS_HOSTED_CONTROL_PLANE=""
 
-NAMESPACE_SUBSCRIPTION="rhdh-operator"
+export NAMESPACE_SUBSCRIPTION="rhdh-operator"
```

Comment:
If this variable is not used, I think you can safely remove it. It should not be needed elsewhere.